autode/utils.py

```diff
@@ -15,6 +15,27 @@
 # Needed for additional pickle-ability
 multiprocessing.set_start_method("fork")
 
+try:
```

Comment:
@josephsilcock ok solution?

Reply:
I'll assume so :joy:

Reply:
ahh sorry I didn't see this, must have got missed in all the emails I got every time you pushed ;)
next time feel free to message me

Reply:
this seems fine
would it be better to check if the version of python is less than 3.8 though? (sys.version_info)

Reply:
yep. code > comments